### ISSUE TITLE
crash on start

### ISSUE THREAD

**Issue by jonorthwash** [goavki/phenny#328]
I get the following crash when begiak starts now:
```
Traceback (most recent call last):
  File "./phenny", line 231, in <module>
    main()
  File "./phenny", line 228, in main
    run(config_module) # @@ thread this
  File "/home/begiak/phenny/__init__.py", line 70, in run
    t.run()
  File "/usr/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/home/begiak/phenny/__init__.py", line 57, in run_phenny
    try: connect(config)
  File "/home/begiak/phenny/__init__.py", line 48, in connect
    p = bot.Phenny(config)
  File "/home/begiak/phenny/bot.py", line 47, in __init__
    self.setup()
  File "/home/begiak/phenny/bot.py", line 95, in setup
    self.bind_commands()
  File "/home/begiak/phenny/bot.py", line 182, in bind_commands
    for name, func in functions.items():
AttributeError: 'function' object has no attribute 'items'
```

**Comment by jonorthwash** [comment]
@Androbin, could you take a look at this?

**Comment by sushain97** [comment]
I reverted https://github.com/goavki/phenny/commit/f5e6ffb48c8f88a97b9081df74bccb9b162fb62c which probably caused this.

**Comment by jonorthwash** [comment]
Seems to be better now, yeah...

**Comment by Androbin** [comment]
I have identified the cause and fixed it in the new PR #330.
